Resources/Prototypes/_RMC14/Entities/Structures/orbital_cannon.yml

```
@@ -96,6 +96,7 @@
 - type: entity
   parent: RMCOrbitalCannonWarheadBase
   id: RMCOrbitalCannonWarheadExplosive
+  description: "An orbital warhead designed to be fired from the UNMC Orbital Cannon system, after sufficient solid fuel has been inserted. This particular model is designed for use in open areas and causes a large circular explosion if it manages to reach the ground."
```

More correct to have the cluster OB description even if we intend to tell players that this may get eaten by walls
```suggestion
  description: "An orbital warhead designed to be fired from the UNMC Orbital Cannon system, after sufficient solid fuel has been inserted. This particular model is designed for use in heavily-fortified areas, and causes a large circular explosion if it manages to reach the ground."
```